services/core/SQLHistorian/config.sqlite

```diff
@@ -1,9 +1,9 @@
 {
-    "agentid": "sqlhistorian-sqlite",
     "connection": {
         "type": "sqlite",
         "params": {
             "database": "~/.volttron/data/historian.sqlite"
         }
-    }
+    },
+    "all_platforms":"true"
```

Comment:
This should be true rather than "true" as "false" would still evaluate to True when evaluating against python.

Reply:
````
Python 2.7.15rc1 (default, Nov 12 2018, 14:31:15) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> bool("false")
True
````